### DISCUSSION TITLE
How to install for Windows (VSCode)

### DISCUSSION THREAD

**Discussion by Antorux** [frePPLe/frepple#679] [Q&A]
Hello, I ran into a problem with local startup frepple in VScode on Windows, I have docker desktop (wsl enabled) installed, when trying to launch a container in docker dekstop, the database waiting error occurred. There are also errors in VS Code, I cloned the repository, created the build folder where I registered cmake.., and then tried to connect to the database, which turned out to be unsuccessful.

**Comment by jdetaeye** [comment]
The development container doesn't include a postgres database. You should spin up your own, external to the container.

This can be either;
- As a separate container.
   There is a  dockerfile in the .devcontainer folder to assist in that
- As a native install on your windows host
   Your development container can then refers to it through the special host name "host.docker.internal" in the "djangosettings.py" file.
- As a separate installation on a separate machine.
   Edit the "djangosettings.py" file in your djangosettings.py file with the correct server, port, user and password.

**Comment by Antorux** [comment]
Okay, there's still a question. I downloaded the old version of frepple for Windows (frepple.exe 6.15 version), installed it along with the database. Frepple runs locally on port 8000 and connects to the database. But how can I see this database (tables, views) in pgAdmin 4 or through the console.

**Comment by jdetaeye** [comment]
>> I downloaded the old version of frepple for Windows (frepple.exe 6.15 version),

Don't do that.  ⛔  
That's an old frepple version and an old postgres version. You're on your own on this track... 

Do it the proper way and install a recent postgres from https://www.postgresql.org/download/windows/ : v14, v15 or v16.  We haven't tried the very new postgres v17 yet.
